manifest.md

```diff
@@ -83,7 +87,7 @@ Unlike the [image index](image-index.md), which contains information about a set
         - [`application/vnd.oci.image.layer.nondistributable.v1.tar+gzip`](layer.md#gzip-media-types)
 
         Manifests concerned with portability SHOULD use one of the above media types.
-        An encountered `mediaType` that is unknown to the implementation MUST be ignored.
+        An encountered `mediaType` that is unknown to the implementation MUST NOT generate an error.
```

Comment:
Image runtimes should be allowed to generate an error if they cannot process the layers. Same for artifact processing tools that encounter an unexpected layer (e.g. helm trying to process a gif layer).

Reply:
How would you phrase that distinction?

Does it make sense for us to define consumers (runtimes) vs distributors (registries, proxies) somewhere?

Reply:
We say "implementations storing or copying content" above. That phrasing could work here too.